docker/images/node/Dockerfile

```diff
@@ -14,18 +13,17 @@ RUN apt-get update \
 
 COPY package.json package.json
 COPY gulpfile.babel.js gulpfile.babel.js
-COPY front/ front
 
 # we use `--no-optional` because some optional npm dependencies fail on install
 RUN npm install -g gulp-cli
 # Took decision from SO: https://goo.gl/3uXZxw
 RUN npm install imagemin-jpegtran imagemin-svgo imagemin-gifsicle imagemin-optipng --save
 RUN npm install --no-optional
 
+COPY front/ front
 # Run it to check if `gulp build` works.
 # There was problems with it. See stb#258 github issue.
-RUN gulp build
+RUN echo $DEPS_DIR > /etc/environment && gulp build
```

Comment:
It'll be good to do `echo $DEPS_DIR > /etc/environment` directly after setting `DEPS_DIR` var.
The whole env definition at top of the file is good practice, i think.

it's not clear how it works: /etc/environment should contain line like this: `DEPS_DIR="/usr/app/deps"`.
But with your code it'll look like this: `/usr/app/deps`